feat: support .redocly.yaml for options for redoc-cli (#1981)
Redocly/redoc: cli/README.md

- Bundle using a custom template and add custom `templateOptions`:<br/>
  `$ redoc-cli build [spec] -t custom.hbs --templateOptions.metaDescription "Page meta description"`

> #### With `.redocly.yaml` file ([more info](https://redocly.com/docs/cli/configuration/#redocly-configuration-file)):

Reviewer: ```suggestion
#### With Redocly configuration file ([more info](https://redocly.com/docs/cli/configuration/#redocly-configuration-file)):
```
(I missed a spot)

> #### With `.redocly.yaml` file ([more info](https://redoc.ly/docs/api-reference-docs/on-premise/cli-usage/#example-with-try-it-console-enabled)):

Author: ```suggestion
#### With `.redocly.yaml` file ([more info](https://redocly.com/docs/cli/configuration/#redocly-configuration-file)):
```

> 1. Go to folder with `.redocly.yaml` and your definition file

Reviewer: Does it work with `redocly.yaml` file too?

Author: Yes, it works too

Reviewer: ```suggestion
1. Go to folder with your Redocly configuration file (`.redocly.yaml` or `redocly.yaml`) and your OpenAPI definition file.
```

> 2. Build site using command (options from `.redocly.yaml` file will be automatically fetched):  

Reviewer: ```suggestion
2. Build the site using the `build` command (options from the Redocly configuration file will be automatically fetched):  
```
